ddl/db_change_test.go

```diff
@@ -877,6 +877,14 @@ func runTestInSchemaState(
 	}
 }
 
+func currentSchemaState(job *model.Job) model.SchemaState {
```

Comment:
It would mix the subJob's schemaState with the whole job's schemaState. The name should be refined

Reply:
How about `jobStateOrLastSubJobState`?